src/templates/user/ico_subscribe.html.tt

```diff
@@ -176,6 +176,10 @@
     <p>[% l("Please select your ICO account to proceed.") %]</p>
 </div>
 
+<div class="invisible" id="ico_account_message_real">
+    <p>[% l("Please select your real account to proceed.") %]</p>
```

Comment:
can we change the wording to real money account

Reply:
Sure